src/pubsub/momento.rs

```diff
@@ -76,26 +76,26 @@ async fn subscriber_task(client: Arc<TopicClient>, cache_name: String, topic: St
         PUBSUB_SUBSCRIBER_CURR.add(1);
         PUBSUB_SUBSCRIBE_OK.increment();
 
-        let msg_stamp = MessageValidator::new();
+        let validator = MessageValidator::new();
 
         while RUNNING.load(Ordering::Relaxed) {
             match subscription.next().await {
                 Some(SubscriptionItem::Value(v)) => {
                     if let ValueKind::Binary(mut v) = v.kind {
-                        match msg_stamp.validate_msg(&mut v) {
-                            MessageValidationResult::Unexpected => {
+                        match validator.validate(&mut v) {
+                            Err(ValidationError::Unexpected) => {
                                 error!("pubsub: invalid message received");
                                 RESPONSE_EX.increment();
                                 PUBSUB_RECEIVE_INVALID.increment();
                                 continue;
                             }
-                            MessageValidationResult::Corrupted => {
+                            Err(ValidationError::Corrupted) => {
```

Comment:
Curious why one of these cases increments RESPONSE_EX and the other PUBSUB_RECEIVE, but both have a message suggesting they are pubsub messages.